exporters/logging/src/test/java/io/opentelemetry/exporters/logging/LoggingSpanExporterTest.java

```diff
@@ -60,8 +60,8 @@ void returnCode() {
     SpanData spanData =
         TestSpanData.newBuilder()
             .setHasEnded(true)
-            .setTraceId(new TraceId(1234L, 6789L))
-            .setSpanId(new SpanId(9876L))
+            .setTraceId(TraceId.fromLongs(1234L, 6789L))
```

Comment:
A note for potentially following up with moving `fromLongs` to ID generation, there isn't a great reason for unit tests to use the long version vs string version, presumably assertion messages matching the input is an improvement.